<launch>
    <!-- Performing Task State -->
    <!-- This launch file defines the nodes active during task execution. -->

    <!-- Node to perform the task -->
    <node pkg="uav_autonomous_utils" type="perform_task_node.py" name="perform_task_node" output="screen" />

</launch>